content/en/docs/concepts/storage/volumes.md

```diff
@@ -254,12 +254,52 @@ storage](/docs/concepts/configuration/manage-resources-containers/#setting-reque
 If that is filled up from another source (for example, log files or image
 overlays), the `emptyDir` may run out of capacity before this limit.
 
+#### Considerations for memory backed `emptyDir` volumes {#emptydir-memory-backed}
+
 {{< note >}}
-If the `SizeMemoryBackedVolumes` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled,
-you can specify a size for memory backed volumes.  If no size is specified, memory
-backed volumes are sized to node allocatable memory.
+You can specify a size for memory backed volumes, provided that the `SizeMemoryBackedVolumes`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+enabled in your cluster (this has been beta, and active by default, since the Kubernetes 1.22 release).
+If you don't specify a volume size, memory backed volumes are sized to node allocatable memory.
 {{< /note>}}
 
+{{< caution >}}
+If you do not specify a `sizeLimit` for `emptyDir`, you can create a
+memory backed `emptyDir` with node allocatable size unintentionally.  Also,
+user can create multiple memory backed `emptyDir`s on the same node, each set
+to a node allocatable size.  And what Kubernetes sees as memory usage is the
+amount that is actually consuming the memory　backed `emptyDir`, so even after
+you create a memory backed `emptyDir` with node allocatable size, the other Pods
+can be scheduled to the same node.  As a result, multiple memory backed
+`emptyDir`s with node allocatable size can be created on the same node.
+{{< /caution >}}
+
+If you use memory backed `emptyDir` without appropriate settings, the following
+risks may occur:
+
+* A single Pod with memory backed `emptyDir` may unintentionally occupy node's memory.
+* If multiple memory backed `emptyDir`s are created on the same mode, their total
```

Comment:
This isn't really a problem with memory-backed empty-dir volumes, right?  If you schedule pods with memory limits higher than memory requests, you can run into this scenario.

Reply:
Yes, ditto. This is also the case when memory request and memory limit are not set, so these are added as the conditions mentioned above.